WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/TopTabView.swift

```diff
@@ -223,6 +228,9 @@ struct TopTabView<Content: View>: View {
                 .frame(height: contentSize.height)
             }
         }
+        .onAppear() {
+            selectedTab = selectedTabIndex ?? 0
```

Comment:
I had a dilemma whether to add a separate `init` parameter to set initially selected tab or reuse `selectedTabIndex` parameter.
I went with this approach to avoid extra changes.

Reply:
Makes sense to me!

nit: Does this need to be set here, or could it be set in the `onAppear` above (lines 130-131) where we handle the rest of the selected tab appearance?

Reply:
> nit: Does this need to be set here, or could it be set in the `onAppear` above (lines 130-131) where we handle the rest of the selected tab appearance?

Good point!